elasticsearch/linux_distributions/opendistro-tar-build.sh

```diff
@@ -26,7 +26,11 @@ S3_RELEASE_FINAL_BUILD=`yq eval '.urls.ODFE.releases_final_build' $MANIFEST_FILE
 S3_RELEASE_BUCKET=`echo $S3_RELEASE_BASEURL | awk -F '/' '{print $3}'`
 PACKAGE_NAME="opendistroforelasticsearch"
 TARGET_DIR="$ROOT/target"
-plugin_version=$OD_VERSION
+PLATFORM="linux"; if [ ! -z "$1" ]; then PLATFORM=$1; fi; echo PLATFORM $PLATFORM
+ARCHITECTURE="x64"; if [ ! -z "$2" ]; then ARCHITECTURE=$2; fi; echo ARCHITECTURE $ARCHITECTURE
+ES_URL=`yq eval '.urls.ES.'$PLATFORM'_'$ARCHITECTURE'' $MANIFEST_FILE`
+ES_ARTIFACT=`echo $ES_URL | awk -F '/' '{print $6}'`
```

Comment:
better use basename in case we change url later.

Reply:
I wanted to avoid including the type in the wget, so went with the full name instead of the base name. In this approach, we control everything from the manifest.